src/lib/util.js

```diff
@@ -74,6 +78,9 @@ module.exports = {
             return;
         }
         fse.removeSync(dirs.in);
+        if (dirExistsAndEmpty(dirs.errors)) {
+            fse.removeSync(dirs.errors);
+        }
```

Comment:
This should be covered by the line below which deletes the parent `out/` folder already. By nature, we want this inside the out folder, so it's also unlikely the errors folder would move somewhere else.

Reply:
looks like it is not needed.